README.md

```diff
@@ -14,7 +13,19 @@
 >
 > - [**The IPFS Network Spec**](https://github.com/ipfs/specs/tree/master/protocol/network), which grew into libp2p
 > - [**go-libp2p implementation**](https://github.com/ipfs/go-libp2p)
-> - [**node-libp2p implementation**](https://github.com/diasdavid/node-libp2p)
+> - [**js-libp2p implementation**](https://github.com/diasdavid/js-libp2p)
+
+# Usage
+
+## Install
+
+```bash
+$ go get github.com/ipfs/go-libp2p
+```
 
-## This README is WIP
+# Run tests
 
+```bash
+$ cd $GOPATH/src/github.com/ipfs/go-libp2p
+$ GO15VENDOREXPERIMENT=1 go test ./p2p/<path of _test.go you want to run>
```

Comment:
sounds good :) pushed on https://github.com/diasdavid/go-libp2p/commit/35a0ea39a32b565fc4a84e9cdcaf777d4000154c, adding everything to the same PR, closing this one.